mailserver/registry/verifier_test.go

```diff
@@ -64,28 +62,12 @@ func (s *VerifierTestSuite) setupAccount() {
 }
 
 func (s *VerifierTestSuite) add(nodeID discover.NodeID) {
-	auth := s.newKeyedTransactor()
+	auth := bind.NewKeyedTransactor(s.privKey)
```

Comment:
my point was that you will have to use signer with chain id for ropsten or mainnet. i dont think that NewKeyedTransactor does that